db/data/regions/NG/ja.yml

```diff
@@ -5,7 +5,7 @@ ja:
       addresses:
         province:
           label:
-            default: 州
+            default: 都道府県/州
             optional: 州 (任意)
```

Comment:
The default and optional labels should be the same, except for the content of the parenthesis.